Source/Foundation/Association.swift

```diff
@@ -55,10 +55,16 @@ public func associatedProperty<T: AnyObject>(host: AnyObject, keyPath: StaticStr
 ///
 /// N.B. Ensure that `host` isn't strongly captured by `initial` or `setter`, otherwise this
 /// will create a retain cycle with `host` causing it to never dealloc.
```

Comment:
This comment and the ones like it below can all be removed now since you've isolated the problem and caller's don't need to worry about it.